src/components/questDisplay.html

```diff
@@ -1,42 +1,41 @@
-        <div class="row justify-content-center">
-            <div class="col-10">
-                <div class="progress"
-                     data-bind='tooltip: {title: ko.pureComputed(function(){return QuestHelper.questList()[player.currentQuest().index].progressText()}), trigger: "hover"}'>
-                    <div class="progress-bar bg-success" role="progressbar"
-                         data-bind="attr:{ style: 'width:' + (QuestHelper.questList()[player.currentQuest().index].progress() * 100) + '%' }"
-                         aria-valuemin="0" aria-valuemax="100">
-                    </div>
-                </div>
-                <div>
-                                <span data-bind='if: QuestHelper.questList()[player.currentQuest().index].isCompleted()'>
-                                    <button class='btn btn-sm btn-success'
-                                            data-bind='click: function(){QuestHelper.questList()[player.currentQuest().index].claimReward()}'>
-                                        Claim
-                                    </button>
-                                </span>
-                    <span>
-                                    <button class='btn btn-sm btn-primary' style="margin:5px"
-                                            data-bind='click: function(){$("#QuestModal").modal("show")}'>
-                                        Questlist
-                                    </button>
-                                </span>
-                    <span data-bind='if: !QuestHelper.questList()[player.currentQuest().index].isCompleted()'>
-                                    <button class='btn btn-sm btn-danger' style="margin:5px"
-                                            data-bind='click: function(){QuestHelper.quitQuest()}'>
-                                        Quit
-                                    </button>
-                                </span>
 
+    <!-- List of quests -->
+    <div data-bind="foreach: player.currentQuests">
+        <div class="card border-bottom-0">
+            <p style="margin-bottom: 0px" data-bind="text: QuestHelper.questList()[$data.index].description"></p>
+            <div class="row justify-content-sm-center">
+                <div class="col-sm-11 col-md-8">
+                    <div class="row" style="margin-bottom: 10px">
+                        <div class="col-9" style="padding:0px; margin:0px">
+                            <div class="progress" style="margin-top:6px; padding:0px; text-align:center" 
+                                data-bind='tooltip: {title: QuestHelper.questList()[$data.index].progressText, trigger: "hover"}'>
+                                <div class="progress-bar bg-success" role="progressbar" 
+                                    data-bind="attr:{style: 'width:' + (QuestHelper.questList()[$data.index].progress() * 100) + '%'}"
+                                    aria-valuemin="0" aria-valuemax="100">
```

Comment:
If you wanted to have the text inside the progress bar similar to how the breeding progress bar is shown, you should be able to do something like this:
```suggestion
                                    aria-valuemin="0" aria-valuemax="100">
                                    <span data-bind="text: Math.floor((QuestHelper.questList()[$data.index].progress() * 100)) + '%'">0%</span>
```
**Note:** The span element will need to be the same width as the full progress bar (80% for the breeding progress bar)
![image](https://user-images.githubusercontent.com/7288322/58848783-e20d6800-86db-11e9-997c-8dc311dd5d31.png)